packages/ra-ui-materialui/src/list/List.js

```diff
@@ -90,8 +90,8 @@ List.propTypes = {
     hasEdit: PropTypes.bool.isRequired,
     hasList: PropTypes.bool.isRequired,
     hasShow: PropTypes.bool.isRequired,
-    location: PropTypes.object.isRequired,
-    match: PropTypes.object.isRequired,
+    location: PropTypes.object, // deprecated
```

Comment:
same

Reply:
I'm not very good with PropTypes. Should I remove it? Won't that make React complain if it is passed in?

Reply:
Don't remove the line, just the comment.